package.json

```diff
@@ -21,6 +21,7 @@
     "axios": "^0.21.4",
     "date-fns": "^2.25.0",
     "formik": "^2.2.9",
+    "monaco-editor": "^0.28.1",
```

Comment:
line 8 we already use @monaco-editor/react. Can you use that as well or should we switch over the other one to use your version?